src/app/resize/resize.component.ts

```diff
@@ -0,0 +1,72 @@
+import {Component, ElementRef, EventEmitter, Input, Output} from "@angular/core";
+
+@Component({
+	selector: 'resize',
```

Comment:
Add a prefix to the component name?
I've been using `app` so far in hopes that I'll rename it at some point. 
see https://angular.io/styleguide#!#02-07